### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Authorization_Keyboard: On mouse-hovering at non-text content at "Use Specific HTTP Methods" label we are getting some information which is inaccessible using keyboard

### ISSUE THREAD

**Issue by InduPriya1805** [Microsoft/IIS.WebManager#124]
GitHubTags: #A11y_Manage.IIS.Net; #A11yMAS; #A11yTCS; #C+E-H; #DesktopWeb; #Win10; #.NETWebsites; #Edge; #Chrome; #Keyboard; #MAS2.1.1

**Environment Details:**
URL: https://manage.iis.net/webserver/authorization
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome 68.0.3440.106
Keyboard
MAS ID Violated: MAS 2.1.1

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till Web Server and press enter
5. Tab till Authorization and press enter
6. Tab till more button (...) and press enter
8. Verify non text content at "Use Specific HTTP Methods" label is accessible through keyboard

**Actual Result:**
Non text content at "Use Specific HTTP Methods" label is not  accessible through keyboard

**Expected Result:**
Non text content at "Use Specific HTTP Methods" label should be accessible through keyboard

**Note:**
This issue is observed in all pages which has non-text content (question mark) that reveals some information regarding the label

**User Impact:**
Users with disabilities could not get the complete information if it is inaccessible using keyboard

**Attachments:**
![bug124_web server_authorization_non text content inaccessible using keyboard](https://user-images.githubusercontent.com/42638450/44707298-c24c7200-aac1-11e8-83d9-7c8c52ac6ef8.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Yashi1201** [comment]
**Github tags:** #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
